src/metabase/async/api_response.clj

```diff
@@ -202,7 +202,15 @@
 
   java.lang.Boolean
   (write-body-to-stream [bool response output-stream]
-    (ring.protocols/write-body-to-stream (str bool) response output-stream)))
+    (ring.protocols/write-body-to-stream (str bool) response output-stream))
+
+  clojure.lang.Keyword
+  (write-body-to-stream [kkey response output-stream]
+    (ring.protocols/write-body-to-stream
+     (if-let  [key-ns (namespace kkey)]
+       (str key-ns "/" (name kkey))
+       (name kkey))
```

Comment:
Clever!! I think I prefer the more explicit code though. I almost went with cond->, too.

Reply:
for future reference, I just learned that we have a `metabase.shared.util/qualified-name` function

Reply:
I looked at that one, but I went with "/" because it seems more intuitive to have keywords translated to e.g. 
`"metabase.shared.util/qualified-name"`
instead of 
`"metabase.shared.util.qualified-name"`

Maybe the qualified name is the way to go? I'm not sure. But at least this can't break anyone since the server never worked (it would 500) when trying to return a keyword through the api.

Reply:
I don't think that's what it does?

```
user=> (metabase.shared.util/qualified-name :metabase.shared.util/qualified-name)
"metabase.shared.util/qualified-name"
```

Reply:
Good call I'll switch it over